### DISCUSSION TITLE
Check out our Beat Battles audio player: wavesurfer.js + three.js 3D visualizer

### DISCUSSION THREAD

**Discussion by b0ss-x** [katspaugh/wavesurfer.js#3352] [Show and tell]
We built a custom audio player for Audeobox NEXXT [Beat Battles](https://www.audeobox.com/battles) using wavesurfer.js and three.js synchronized 3D visualizer. The video is from the [final battle of a beat battle tournament](https://www.audeobox.com/battles/don-luchini-vs-poizxne-36673/) we recently hosted.

[audeobox-beat-battles-player.webm](https://github.com/katspaugh/wavesurfer.js/assets/150304455/84d25933-7671-4f38-a002-ce511be64b26)

**Comment by katspaugh** [comment]
Love it, thanks for sharing!
